index.js

```diff
@@ -98,6 +106,12 @@ module.exports = {
   // Until we have a better solution for this use case upstream, we are fixing this here by moving the generated
   // files outside of this folder and deleting it
   async moveFilesForExistingMonorepo(options) {
+    assert(
```

Comment:
these asserts weren't needed, but since it can be hard to know what functions have ran by analyzing the output, I figured I'd be a little defensive here. If something did wrong, us addon-blueprint maintainers would be alerted, hopefully

Reply:
Love it